collections/_components/button.html

```diff
@@ -1,18 +1,231 @@
+$btn-disabled-opacity:        .65 !default;
+$btn-active-box-shadow:       inset 0 3px 5px rgba($black, .125) !default;
+
+$btn-link-color:              $link-color !default;
+$btn-link-hover-color:        $link-hover-color !default;
+$btn-link-disabled-color:     $gray-600 !default;
+
+
+$btn-border-radius:           $border-radius !default;
+$btn-border-radius-sm:        $border-radius-sm !default;
+$btn-border-radius-lg:        $border-radius-lg !default;
+
+$btn-transition:              color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out !default;
+
+$btn-hover-bg-shade-amount:       15% !default;
+$btn-hover-bg-tint-amount:        15% !default;
+$btn-hover-border-shade-amount:   20% !default;
+$btn-hover-border-tint-amount:    10% !default;
+$btn-active-bg-shade-amount:      20% !default;
+$btn-active-bg-tint-amount:       20% !default;
+$btn-active-border-shade-amount:  25% !default;
+$btn-active-border-tint-amount:   10% !default;
+{% endcapture %}
+
+{% capture button-default %}
+<button type="button" class="btn btn-primary sgds">Default</button>
+{% endcapture %}
+
+{% capture button-variants %}
+<button type="button" class="m-1 btn btn-primary sgds">primary</button>
+<button type="button" class="m-1 btn btn-secondary sgds">secondary</button>
+<button type="button" class="m-1 btn btn-success sgds">success</button>
+<button type="button" class="m-1 btn btn-danger sgds">danger</button>
+<button type="button" class="m-1 btn btn-warning sgds">warning</button>
+<button type="button" class="m-1 btn btn-info sgds">info</button>
+<button type="button" class="m-1 btn btn-light sgds">light</button>
+<button type="button" class="m-1 btn btn-dark sgds">dark</button>
+{% endcapture %}
+
+{% capture button-outline %}
+<button type="button" class="btn btn-outline-primary sgds">Primary</button>
+<button type="button" class="btn btn-outline-secondary sgds">Secondary</button>
+<button type="button" class="btn btn-outline-success sgds">Success</button>
+<button type="button" class="btn btn-outline-warning sgds">Warning</button>
+<button type="button" class="btn btn-outline-danger sgds">Danger</button>
+<button type="button" class="btn btn-outline-info sgds">Info</button>
+<button type="button" class="btn btn-outline-light sgds">Light</button>
+<button type="button" class="btn btn-outline-dark sgds">Dark</button>
+{% endcapture %}
+
+{% capture button-size %}
+<button type="button" class="btn btn-primary btn-lg sgds">Large button</button>
+<button type="button" class="btn btn-primary btn-sm sgds">Small button</button>
+{% endcapture %}
+
+{% capture button-block %}
+<div class="d-grid gap-2">
+    <button class="btn btn-primary btn-lg" type="button">Button</button>
+    <button class="btn btn-outline-dark btn-lg" type="button">Button</button>
+  </div>
+{% endcapture %}
+
+{% capture button-disable %}
+    <button disabled="" type="button" class="btn btn-primary sgds">Primary button</button>
```

Comment:
remove the equal and quotation marks for disabled attribute
```suggestion
    <button disabled type="button" class="btn btn-primary sgds">Primary button</button>
```